### ISSUE TITLE
When I finetuning with pretrained VITS model on CSMSC, I failed to change the learning rate.

### ISSUE THREAD

**Issue by Mortyzhou-Shef-BIT** [espnet/espnet#4335]
When I finetuning with **pretrained VITS model** on CSMSC

Firstly, I set lr to 1.995e-05, I warmuped it and got a new finetuned model.

Then I stopped training and set learning rate to 1e-4. 
![image](https://user-images.githubusercontent.com/54799259/166802616-f3593511-4d77-4898-866f-d8dbb2828f6b.png)

But the learning rate was failed to change in continued finetuning and it still was 1.995e-05
![image](https://user-images.githubusercontent.com/54799259/166802501-ca279357-e8ae-4fc7-9468-94c20eeca880.png)

Does any method or config to change the learning rate after fine-tuning?

**Comment by Fhrozen** [comment]
You are comparing the lr of optim1 (generator) with the optim2 (discriminator)
For the generator, you should modify optim1: https://github.com/espnet/espnet/blob/de624ed58953d17907fb241c5cb6514f27510162/egs2/csmsc/tts1/conf/tuning/train_full_band_vits.yaml#L136-L145

For the discriminator, you need to modify optim2, and check optim2, no optim1:
https://github.com/espnet/espnet/blob/de624ed58953d17907fb241c5cb6514f27510162/egs2/csmsc/tts1/conf/tuning/train_full_band_vits.yaml#L146-L155

Also, remember that both optimizers are employing exponential schedulers, which will change the value of lr. If you want to get a fixed lr value, then disable the schedulers.

**Comment by Mortyzhou-Shef-BIT** [comment]
> You are comparing the lr of optim1 (generator) with the optim2 (discriminator) For the generator, you should modify optim1:
> 
> https://github.com/espnet/espnet/blob/de624ed58953d17907fb241c5cb6514f27510162/egs2/csmsc/tts1/conf/tuning/train_full_band_vits.yaml#L136-L145
> 
> For the discriminator, you need to modify optim2, and check optim2, no optim1:
> 
> https://github.com/espnet/espnet/blob/de624ed58953d17907fb241c5cb6514f27510162/egs2/csmsc/tts1/conf/tuning/train_full_band_vits.yaml#L146-L155
> 
> Also, remember that both optimizers are employing exponential schedulers, which will change the value of lr. If you want to get a fixed lr value, then disable the schedulers.

Thank U for your help. I am sorry to give a wrong cut. So I re-defined my description:
**I changed the Optimizer1 and Optimizer2 to same learning rate.**
I finetuned the model with Optimizer1 and Optimizer2's lr = 2e-5  **firstly**

I stopped training and I want to used higher learning rate to finetuning the G and D. 

Then I reset the learning rate of Optimizer1 and Optimizer2 to  lr = 4e-4 with config yaml file.
![image](https://user-images.githubusercontent.com/54799259/167145129-8a2cd4e2-f612-4a5c-bd74-68fb2a771327.png)

However, in the training log. the config log showed Optimizer1 and Optimizer2's lr = 4e-4. But during the training, the learning rate of  Optimizer1 and Optimizer2 still was the initial rate: 2e-5, the cut in the same train log is below

`2022-05-03 23:00:43,292 (gan_trainer:305) INFO: 11epoch:train:101-150batch: iter_time=2.571e-04, discriminator_forward_time=1.713, discriminator_loss=2.168, discriminator_real_loss=1.116, discriminator_fake_loss=1.052, discriminator_backward_time=1.371, discriminator_optim_step_time=0.020, **optim1_lr0=1.998e-05**, discriminator_train_time=3.158, generator_forward_time=0.462, generator_loss=33.407, generator_mel_loss=18.065, generator_kl_loss=1.720, generator_dur_loss=1.910, generator_adv_loss=2.667, generator_feat_match_loss=9.045, generator_backward_time=2.001, generator_optim_step_time=0.133, **optim0_lr0=1.998e-05**, generator_train_time=2.675, train_time=5.903`

I reset the learning rate with different rate, but the learning rate of them are not changed.
![image](https://user-images.githubusercontent.com/54799259/167146032-9d8acddb-c1fa-4dd8-92d2-d85a432d9dd1.png)
![image](https://user-images.githubusercontent.com/54799259/167146074-4218abf1-e6a3-4d45-958e-5ec464a0e3ce.png)
